maps/away/ships/coc/gadpathur_patrol/gadpathur_patrol.dm

```diff
@@ -2,7 +2,7 @@
 	name = "Gadpathurian Patrol Corvette"
 	description = "Gadpathur navy patrol ship."
 	suffixes = list("ships/coc/gadpathur_patrol/gadpathur_patrol.dmm")
-	sectors = list(ALL_COALITION_SECTORS) //NOTE: Gadpathur patrols all of the Coalition, however, they are intentionally -not- present in Haneunim. Konyang and Gadpathur are not friendly as of the Amor Patriae arc.
+	sectors = list(SECTOR_WEEPING_STARS, SECTOR_COALITION, SECTOR_LIBERTYS_CRADLE, SECTOR_ARUSHA) //NOTE: Gadpathur patrols all of the Coalition, however, they are intentionally -not- present in Haneunim. Konyang and Gadpathur are not friendly as of the Amor Patriae arc.
```

Comment:
Do this via the sectors_blacklist var added in <https://github.com/Aurorastation/Aurora.3/pull/17733>

Reply:
For example here keep it `sectors = list(ALL_COALITION_SECTORS)`
but add `sectors_blacklist = list(HEUMENIM, BURSIZIA)` or however they're spelled

I added the blacklist cause otherwise it's ass to have to go through all old away sites and ships when a new sector is added